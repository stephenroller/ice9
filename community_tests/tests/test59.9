# break w/in do

var i : int

i := 0;

do i < 5 ->
	if i = 3 ->
		break;
	fi
	write i;
	i := i + 1;
od

write 6;
