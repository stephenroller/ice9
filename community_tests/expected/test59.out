Loading file: generated/test59.tm
0 
1 
2 
6 
Number of instructions executed = 191
